.travis.yml

```diff
@@ -17,6 +17,14 @@ install:
   - pip3 install -r requirements.txt
   - python3 setup.py --getdeps --log
   - pip3 install . --upgrade
+  - if [[ -z "$TRAVIS_TAG" ]]; then exit 0; fi
+  - echo "[distutils]" > $HOME/.pypirc
```

Comment:
Maybe we can host the header of this file in the repo and add only username/password here?

Reply:
We could, but it's very small so this makes it one unrelated file less.